app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, current_file, new_file = command.split()
+    if current_file != new_file:
+        with (open(current_file, "r") as file_in,
+              open(new_file, "w") as file_out):
+            file_out.write(file_in.read())
```

Comment:
Please, add checking that command is correct:
1. Starts from cp
2. Contains all arguments